### PR TITLE
Alert component redesign

### PR DESCRIPTION
**Pull request by andresgutgon**
# What?
Change styles on our `<Alert />` component to much more Latitude design system.

### TODO
- [x] Copy styles from old `<Notification />` component
- [x] Make sure a chart error looks good

image

### CODE REVIEW

File: packages/client/svelte/src/lib/ui/alert/alert.svelte [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
+  export let secondary: $$Props['secondary'] = false
   export { className as class }
+
+  $: styles = theme.ui.alert.cssClass({
+    type,
+    scrollable,
+    secondary,
+    className,
+  })
+  $: textColor = styles.properties.foreground
```
Comment (andresgutgon) [inline packages/client/svelte/src/lib/ui/alert/alert.svelte:33]: If the prop `type` change `cssClass` compute new `background` and `foreground` colors. But `textColors` is not updated. Any idea why?

File: packages/client/svelte/src/lib/ui/alert/alert.svelte [hunk 2]
```diff
@@ -1,23 +1,45 @@
...
   import { theme } from '@latitude-data/client'
-  import type { HTMLAttributes } from 'svelte/elements'
+  import AlertText from './text.svelte'
 
-  type $$Props = HTMLAttributes<HTMLDivElement> & theme.ui.alert.Props
+  type $$Props = Props
 
   let className: $$Props['class'] = undefined
-  export let variant: $$Props['variant'] = 'default'
+  export let type: $$Props['type']
```
Comment (csansoon) [inline packages/client/svelte/src/lib/ui/alert/alert.svelte:20]: Update the Docs with the changes on API

**Review by csansoon — DISMISSED** [review]

File: packages/client/svelte/src/lib/ui/alert/alert.svelte (hunk 2 above)
Comment (andresgutgon) [reply packages/client/svelte/src/lib/ui/alert/alert.svelte:20]: Good catch 💯

File: packages/client/svelte/src/lib/ui/alert/alert.svelte (hunk 1 above)
Comment (geclos) [reply packages/client/svelte/src/lib/ui/alert/alert.svelte:34]: :\

**Pull request merged**